docs/api.md

```diff
@@ -295,6 +306,26 @@ Visibility: Private
       ]
     }
 
+### Delete OAuth account by user id
```

Comment:
these docs may need a quick update to match the changes in the `delete_account_oauth` handler

Reply:
Thanks @cainlevy, doc updated 😄